examples/third_party/libpng/BUILD.libpng.bazel

```diff
@@ -10,6 +10,7 @@ filegroup(
 cmake(
     name = "libpng",
     cache_entries = {
+        "CMAKE_POLICY_DEFAULT_CMP0074": "NEW",
```

Comment:
What's this for?

Reply:
This enables use of the _ROOT variables in CMake. Unfortunately it was still failing to find zlib correctly on RBE. I suspect the non-RBE builds are finding a system install of zlib.
ZLIB_ROOT probably needs to be `$(location @zlib)` to get this build working on RBE